models/articles.models.js

```diff
@@ -120,3 +120,39 @@ exports.updateVotes = (article_id, inc_votes) => {
       return rows;
     });
 };
+
+exports.checkCommentExists = (comment_id) => {
+  return db
+    .query(
+      `
+      SELECT *
+      FROM comments
+      WHERE comment_id = $1`,
```

Comment:
Great job - but we dont need this first query. If in the delete query we were returning * we would know if it existed in the first place :)